### PR TITLE
Remote notifications added to Permission Controller

### PR DESCRIPTION
**Pull request by kovalandrew**
- New Permission type added
- Actual iOS/Android implementation added for Remote Notifications
- Android sample project build error fixed
- Sample project updated with using local Maven

### CODE REVIEW

File: permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt [hunk 1]
```diff
@@ -80,6 +81,13 @@ actual class PermissionsController(
                 Manifest.permission.BLUETOOTH,
                 Manifest.permission.ACCESS_COARSE_LOCATION
             )
+            Permission.REMOTE_NOTIFICATION -> {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                    listOf(Manifest.permission.ACCESS_NOTIFICATION_POLICY)
```
Comment (Alex009) [inline permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt]: Android not require runtime permissions for notifications. Just return empty list. Current version request incorrect permission - it access to change notifications of other apps

File: settings.gradle.kts [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 pluginManagement {
     repositories {
+        mavenLocal()
```
Comment (Alex009) [inline settings.gradle.kts]: For what this added?

File: permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt (hunk 1 above)
Comment (kovalandrew) [reply permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt]: Ok, will change with Permission.REMOTE_NOTIFICATION -> emptyList()

File: settings.gradle.kts (hunk 2 above)
Comment (kovalandrew) [reply settings.gradle.kts]: For changes testing without version release.

File: permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt [hunk 3]
```diff
@@ -7,6 +7,7 @@ package dev.icerock.moko.permissions
 import android.Manifest
 import android.content.Context
 import android.content.pm.PackageManager
+import android.os.Build
```
Comment (Alex009) [inline permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt]: not used import

Comment (kovalandrew) [reply permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsController.kt]: removed

**Comment by kovalandrew** [comment]
Readme updated with new type

**Pull request merged**